service/frontend/namespace_handler.go

```diff
@@ -561,6 +561,7 @@ func (d *namespaceHandlerImpl) UpdateNamespace(
 		if updateReplicationConfig.GetActiveClusterName() != "" {
 			activeClusterChanged = true
 			replicationConfig.ActiveClusterName = updateReplicationConfig.GetActiveClusterName()
+			replicationConfig.State = enumspb.REPLICATION_STATE_NORMAL
```

Comment:
you might also have this in namespace replication task executor

Reply:
what do you mean?

Reply:
So if for whatever reason active cluster is not changed but this status is set, we cannot unset it?

Reply:
or we can only failover to unset the state var. 

i guess the lifecycle is not entire clear, let us talk

Reply:
I mean here: https://github.com/temporalio/temporal/blob/4b4dbf3978ed2df5e25792fec9ca5524a1647080/common/namespace/replication_task_executor.go#L296-L302. If the namespace is in handover state and the namespace is failed over from another cluster. Then the replication path should also reset the state.